CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 5.1.5 (2022-01-03)
 
 - Added support for wrapped text in custom components. [#176](https://github.com/blackbaud/skyux-ag-grid/pull/176)
+- Fixed an error with the validator popover change detection. [#180](https://github.com/blackbaud/skyux-ag-grid/pull/180)
```

Comment:
Why do we have `popover` in there? I checked https://github.com/blackbaud/skyux-ag-grid/pull/180/files, and I don't see any connection to the popover component. Is this correct? I assume it's actually referring to a popover element within the ag-grid module, but I'm not sure. FWIW, I think it reads awkwardly with `popover` in there, and unless that point is really important to call out, I'd consider removing `popover` for clarity. ... I'll go ahead and approve the PR though in case we do need `popover` in there. If not and you remove it, let me know, and I'll approve again.